discovery-provider/plugins/notifications/src/logger.ts

```diff
@@ -1,8 +1,8 @@
-import pino, { stdTimeFunctions } from 'pino'
+import pino, { stdTimeFunctions, levels } from 'pino'
 
 export const logger = pino({
   name: `notifications`,
   base: undefined,
-  timestamp: stdTimeFunctions.isoTime,
-  level: process.env.NODE_ENV === 'test' ? 'error' : undefined
+  timestamp: stdTimeFunctions.isoTime
+  // level: process.env.NODE_ENV === 'test' ? 'error' : levels
```

Comment:
can we delete this line if not needed? and rm the levels import if it's not needed?